var/spack/repos/builtin/packages/py-pydot2/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPydot2(PythonPackage):
+    """Python interface to Graphviz's Dot"""
+
+    homepage = "https://pypi.org/project/pydot2/"
+    url      = "https://pypi.io/packages/source/p/pydot2/pydot2-1.0.33.tar.gz"
+
+    version('1.0.33', sha256='02c0e681a1c437077e2bb2522fb81fa322e53ba7002cfda8b894db0392a1bc9b')
+
+    depends_on('py-setuptools', type='build')
+    depends_on('py-pyparsing', type=('build', 'run'))
+
+    conflicts('python@3.5.0:')
```

Comment:
I would remove this. The package does not explicitly say that Python 3.5+ doesn't work. It's likely that the package hasn't been tested on Python 3.5+, and hasn't been updated since Python 3.4 came out, but that doesn't mean it won't work.